scene/gui/code_edit.cpp

```diff
@@ -3139,11 +3139,34 @@ void CodeEdit::_update_scroll_selected_line(float p_mouse_y) {
 	code_completion_force_item_center = -1;
 }
 
+/* Assumes both the new_options and the code_completion_options are sorted. */
```

Comment:
We don't use block comments, even if the comment itself is multiline. So please change all comments to use `//` (and in case of very long comments please wrap them manually at a reasonable breakpoint (the wrapping point is usually around 80 characters).